index.html

```diff
@@ -0,0 +1,161 @@
+            <div class="info-content">
+                <div class="block-title">About the shelter <br> “Cozy House”</div>
+                <p>Currently we have 121 dogs and 342 cats on our hands and statistics show that only 20% of them will
+                    find a family. The others will continue to live with us and will be waiting for a lucky chance to
+                    become dearly loved. </p>
+                <p>We feed our wards with the best food and make sure that they do not get sick, feel comfortable
+                    (including psychologically) and well. We are supported by 87 volunteers and 28 employees of various
+                    skill levels. About 12% of the animals are taken by the shelter staff. Taking care of the animals,
+                    they become attached to the pets and would hardly ever leave them alone.</p>
+            </div>
+        </div>
+        <div class="catalog">
+            <p class="block-title tac mx-auto">Our friends who are looking for a house</p>
+            <div class="pets">
+                <div class="petcard">
+                    <img src="./img/pets-katrine.png" alt="katrine">
+                    <p>Katrine</p>
+                    <a href="">Learn more</a>
+                </div>
+                <div class="petcard">
+                    <img src="./img/pets-jennifer.png" alt="jennifer">
+                    <p>Jennifer</p>
+                    <a href="">Learn more</a>
+                </div>
+                <div class="petcard">
+                    <img src="./img/pets-woody.png" alt="woody">
+                    <p>Woody</p>
+                    <a href="">Learn more</a>
+                </div>
+            </div>
+            <button class="button d-block mx-auto mt-60 mb-100">Get to know the rest</button>
+        </div>
+        <div class="help">
+            <p class="block-title tac mx-auto">
+                How you can help our shelter
+            </p>
+            <div class="itemsfrow">
+                <div class="card">
+                    <img src="./img/icon-pet-food.png" alt="">
+                    <p>Pet food</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-transportation.png" alt="">
+                    <p>Transportation</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-toys.png" alt="">
+                    <p>Toys</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-bowls-and-cups.png" alt="">
+                    <p>Bowls and cups</p>
+                </div>
+                <div class="card">
+                    <img src="./img/icon-shampoos.png" alt="">
+                    <p>Shampoos</p>
+                </div>
+            </div>
+            <div class="itemssrow">
+                <div class="card">
+                    <img src="./img/icon-vitamins.png" alt="">
```

Comment:
```suggestion
                    <img src="./img/icon-vitamins.png" alt="Vitamins">
```

Пустые альты не валидны
закидывай что угодно, но пустой не оставляй

Заголовок тот же карточки скопируй в альт

просто когда закинем сайт в валидатор - и так будет хватать что поправить, а с такими исправлениями будет уж шибко страшный список :D

+ это еще для людей с плохим зрением, кто пользует скринридер
или у кого очень грустный интернет и фото не подгрузилось - будет показан альт